linera-views/src/rocksdb.rs

```diff
@@ -180,6 +189,7 @@ pub type RocksdbContext<E> = ContextFromDb<E, RocksdbClient>;
 #[async_trait]
 impl KeyValueStoreClient for RocksdbClient {
     const MAX_CONNECTIONS: usize = 1;
+    const MAX_VALUE_SIZE: usize = MAX_VALUE_SIZE;
```

Comment:
Shouldn't this be `usize::MAX`, or rather `KeyValueStoreClientBigValue::MAX_VALUE_SIZE`?

Reply:
Yes, `usize::MAX`.